lib/fog/aws/dns.rb

```diff
@@ -3,6 +3,18 @@ module DNS
     class AWS < Fog::Service
       extend Fog::AWS::CredentialFetcher::ServiceMethods
 
+      require 'fog/aws/requests/dns/change_resource_record_sets'
```

Comment:
These should be redundant to the `request` calls that come a bit later (though a couple new ones should be added). By using `request` the requires are deferred, however, until initialization time. So, I think we just want the request versions and not these explicit requires.